content/inferno/time-series/time-series.md

```diff
@@ -0,0 +1,309 @@
+
+- [Modern Pandas](https://tomaugspurger.github.io/modern-7-timeseries)
+
+## Time Series Problems Overview
+
+In the next sections, we see what are the main problems that can be solved with time series data.
+
+::: warning
+Be aware that this is not an exhaustive list, and you should consider it only _a starting point!_ 
+:::
+
+Once you have identified a problem that you are interested in solving, you need to find papers that talk about the state of the art of that problem and go deeper!
+
+It is always true that there is not a silver bullet that solves every problem, remember the [No Free Lunch Theorem](https://en.wikipedia.org/wiki/No_free_lunch_theorem).
+
+Learning to manipulate time data with Pandas is mandatory, and the Pandas DataFrame structure is the natural landing place for this type of data. However, due to the "sequential" nature of the data type, the DataFrame has some structural limitations, because it is designed for more classic tabular data. 
+
+To overcome these limitations, the [Alan Turing Institute](https://www.turing.ac.uk/) has developed a special version of Scikit-Learn, [Sktime](https://github.com/alan-turing-institute/sktime), using a specific .ts format to load time series data into Pandas DataFrames. 
+
+From the docs: 
+> sktime is a Python machine learning toolbox for time series with a unified interface for multiple learning tasks. 
+>sktime provides dedicated time series algorithms and scikit-learn compatible tools for building, tuning, and evaluating composite models. 
+>
+>We currently support:
+>- Forecasting
+>- Time series classification
+>- Time series regression
+>
+>For deep learning methods, see our companion package: [sktime-dl](https://github.com/sktime/sktime-dl).
+
+To get started with SKtime get through the following resourse:
+
+- [SKTime - How to get started](https://www.sktime.org/en/latest/how_to_get_started.html)
+- [Loading data in SKTime](https://www.sktime.org/en/latest/examples/loading_data.html)
+- [SKTime - Examples](https://github.com/alan-turing-institute/sktime/tree/master/examples)
+
+Also, be aware of [TSLearn](https://github.com/tslearn-team/tslearn), a very good machine learning toolkit for time series analysis in Python.
+
+These are the basic tools you need to know to work with time series in Python, let's now have a look at _what type of problem_ you can solve with time series data.
+
+## Time Series Analysis
+
+Imagine you are working as a Data Scientist, and you are asked to do _"something useful"_ not better specified. Your first intuition must be, as with any type of data, to understand the dataset and the type of information it brings with it.
+
+This process, often called *"analysis"*, is fundamental before any other, for example before trying to make predictions about future values.
+
+The path Virgilio suggests is as follows: 
+
+First of all, take this free and extremely complete course that introduces you to time series analysis:
```

Comment:
Also good to mention about Time-series EDA here, see this [Time Series Modelling and Analysis](https://www.youtube.com/playlist?list=PL3N9eeOlCrP5cK0QRQxeJd6GrQvhAtpBK) and [Time-series EDA](https://www.harshaash.com/time-series-eda/)